docs/src/pages/components/radiogroupfield/examples/LabelPositionExample.tsx

```diff
@@ -0,0 +1,35 @@
+import { Radio, RadioGroupField, Flex } from '@aws-amplify/ui-react';
+
+export const LabelPositionExample = () => (
+  <Flex direction="row">
+    <RadioGroupField label="Language" name="language" defaultValue="html">
+      <Radio value="html">html</Radio>
```

Comment:
I see we use strings directly in our codebase, I just wonder if we have considered i18n in future? Should we gradually start using variables instead?

Reply:
This is a good callout. Something we should think about going forward

Reply:
There is an asana task to investigate i18n for primitives if you want to take it 😉

Reply:
I would also callout that i18n for primitives versus docs might be different.